File: an2_sem4/MPP/Homework/sapt9/java/service/src/main/java/ro/ubbcluj/cs/stefnmubb/festivalsellpoint/service/ISpectacolService.java
package ro.ubbcluj.cs.stefnmubb.festivalsellpoint.service;

import ro.ubbcluj.cs.stefnmubb.festivalsellpoint.domain.Spectacol;
import ro.ubbcluj.cs.stefnmubb.festivalsellpoint.repo.EntityRepoException;

import java.time.LocalDateTime;

public interface ISpectacolService extends IService<Integer, Spectacol> {
    Iterable<Spectacol> getBetweenDates(LocalDateTime start, LocalDateTime end) throws EntityRepoException;
}

File: an2_sem4/MPP/Homework/sapt9/cs/FestivalSellpoint/FestivalSellpoint.Service/ISpectacolService.cs
using FestivalSellpoint.Domain;
using System.Collections.Generic;
using System;

namespace FestivalSellpoint.Service
{
    public interface ISpectacolService : IService<int, Spectacol>
    {
        IEnumerable<Spectacol> GetBetweenDates(DateTime start, DateTime end);
    }
}
